src/App.tsx

```diff
@@ -413,6 +427,10 @@ function App(): ReactElement {
   const disableUi: boolean = isRecording || !datasetOpen;
   const disableTimeControlsUi = disableUi;
 
+  let colorRamp = colorRampData.get(colorRampKey)?.colorRamp;
+  if (colorRampReversed && colorRamp) {
+    colorRamp = colorRamp.reverse();
+  }
```

Comment:
I feel like more of this could be done by the ColorRamp class. Maybe a static function that takes in `colorRampKey` and `reversed` and does this logic in there

Reply:
it would also make this more testable. it looks like you have tests for the ui, but not for the ColorRamp class

Reply:
Fixed, thank you!